test-case/manual/DPMST.md

```diff
@@ -0,0 +1,67 @@
+# DPMST test
+DisplayPort Multi-Stream Transport is a manual test.
+
+# Preconditions
+1. Hardware requirements met for each DPMST case
+
+# Test Description
+* Test output from one test unit to two separate monitors with audio output via
+  DP (as well as with USB-C and HDMI adapters if monitors have those inputs
+  available).
+
+# 1) Single DP -> Multi DP hub Test Case:
+## Hardware Requirements
+1. One DisplayPort hub = single DP -> multi-DP
+2. Two monitors with DP input and audio output
```

Comment:
DP-MST will have maximum to 3 DP output. So change to 3 would be better.

Reply:
@xiulipan are you sure? @ClarexZhou indicated the hub they were using was 1->4. 

I will however update this to include testing with more monitors if possible.

Reply:
@stripes416 I just do a quick search on Amazon and found a lot of types of DP-MST hub with different output. So I think either will OK, but it must to be DP-MST hub.

Reply:
Addressed in latest update.